env-hooks/1.ros_distro.sh.em

```diff
@@ -1,13 +0,0 @@
-# generated from ros_environment/env-hooks/1.ros_distro.sh.em
-
-@{
-# This is a build-time environment variable which allows a build engineer to override the expected
-# ROS_DISTRO value for a workspace, for example to deliberately use a newer version of roslib with
-# an older release or vice-versa, or to define a custom distro (eg, "ROS Banana").
-from os import environ
-ROS_DISTRO = environ.get("ROS_DISTRO_OVERRIDE", "TODO")
```

Comment:
What was `ROS_DISTRO_OVERRIDE` used for in ROS 1?

Reply:
> for example to deliberately use a newer version of roslib with an older release or vice-versa

Since the environment hooks are not bundled with `roslib` anymore this use won't exist anymore.

> or to define a custom distro

I added 479033e4bfbf8383d5b6d8ef6f0e9c87272084e3 to keep supporting this corner case.